test/common/http/access_log/access_log_formatter_test.cc

```diff
@@ -128,18 +129,29 @@ TEST(AccessLogFormatterTest, requestInfoFormatter) {
 
   {
     RequestInfoFormatter upstream_format("UPSTREAM_HOST");
-    std::shared_ptr<Upstream::MockHostDescription> host(new Upstream::MockHostDescription());
-    EXPECT_CALL(requestInfo, upstreamHost()).WillRepeatedly(Return(host));
     const std::string host_url = "name";
-    EXPECT_CALL(*host, url()).WillOnce(ReturnRef(host_url));
+    EXPECT_CALL(*requestInfo.host_, url()).WillOnce(ReturnRef(host_url));
```

Comment:
You didn't write this, but as long as you are in here, can you s/requestInfo/request_info.